sky/spot/dashboard/dashboard.py

```diff
@@ -0,0 +1,66 @@
+"""Dashboard for spot jobs based on Flask."""
```

Comment:
Maybe we need a `TODO` here for changing the underlying implementation into a more flexible web programming language like ts or js. Reference: https://github.com/ray-project/ray/tree/master/dashboard/client/src

Reply:
Added.